# System version information in user readable string format

uint64 timestamp		# time since system start (microseconds)
char[32] hw_version		# main autopilot hw version
char[32] sw_version		# main autopilot sw version
char[32] os_version		# OS or middleware version
char[32] bl_version		# bootloader version
char[32] component_version1	# vendor specific component version
char[32] component_version2	# vendor specific component version
